docs/using-hive.md

```diff
@@ -11,6 +11,10 @@
         ```bash
         oc create secret generic mycluster-pull-secret --from-file=.dockerconfigjson=/path/to/pull-secret --type=kubernetes.io/dockerconfigjson
         ```
+  1. Create a Kubenertes secret containing a ssh public key (typically generated with `ssh-keygen`)
+        ```bash
+        oc create secret generic mycluster-ssh-key --from-file=ssh-publickey=/path/to/public/key
+        ```
```

Comment:
Could you also add --from-file=ssh-privatekey=/path/to/private/key. Only specifying public should configure the nodes so you can ssh to them, but if you also specify private you gain automatic fetching of logs from the nodes in the event of an install failure.

Could you also note that this step is optional, and can be done automatically with hiveutil create-cluster via the --ssh-public-key-file and --ssh-private-key-file arguments.

Reply:
Added a new commit with your suggestions.